keyboards/latin47ble/rules.mk

```diff
@@ -0,0 +1,45 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = caterina
+F_CPU = 8000000
+
+
+# Feather uses the caterina bootloader
+OPT_DEFS += -DBOOTLOADER_SIZE=4096 -DCATERINA_BOOTLOADER
+
+
+
+
+# Build Options
+#   change yes to no to disable
+#
```

Comment:
This isn't needed, as the bootloader option sets both of these. 
 So this is redundant. 

```suggestion
```